### DISCUSSION TITLE
TuringCourses/major_basic/

### DISCUSSION THREAD

**Discussion by giscus** [ZJU-Turing/TuringCourses#101] [General]
# TuringCourses/major_basic/

图灵班学习指南

https://zju-turing.github.io/TuringCourses/major_basic/

**Comment by HobbitQia** [comment]
需要注意的是，自 23 级开始，培养方案终于有了修改：AI 专业不再有计算机逻辑设计基础、计算机组成与设计、计算机体系结构和操作系统，而是和信安一样上计算机系统123。而且计逻、计组、体系结构、数据结构基础、高级数据结构与算法分析均提前了一个学期。
